<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeCalculatorDialog</class>
 <widget class="QDialog" name="VolumeCalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>295</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>497</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="glUpSurface">
     <item row="0" column="0">
      <widget class="QLabel" name="lUpSurface">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Upper surface</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rbUpSurfaceMesh">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Mesh</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">bgUpSurface</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cbUpSurfaceMesh"/>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="rbUpSurfacePlan">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Plan</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">bgUpSurface</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="dsUpSurfacePlan">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="glLoSurface">
     <item row="0" column="0">
      <widget class="QLabel" name="lLoSurface">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Lower surface</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rbLoSurfaceMesh">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="locale">
        <locale language="French" country="France"/>
       </property>
       <property name="text">
        <string>Mesh</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">bgLoSurface</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cbLoSurfaceMesh"/>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="rbLoSurfacePlan">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Plan</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">bgLoSurface</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="dsLoSurfacePlan">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hlCalculatesVolume">
     <item>
      <spacer name="hsCalculatesLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCalculates">
       <property name="text">
        <string>Calculates volume</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsCalculatesRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lProcessName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="glResults">
     <item row="0" column="0">
      <widget class="QLabel" name="lSurface">
       <property name="text">
        <string>Surface :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lVolume">
       <property name="text">
        <string>Volume :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VolumeCalculatorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VolumeCalculatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUpSurfaceMesh</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbUpSurfaceMesh</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbUpSurfacePlan</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsUpSurfacePlan</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLoSurfaceMesh</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbLoSurfaceMesh</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLoSurfacePlan</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsLoSurfacePlan</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCalculates</sender>
   <signal>clicked()</signal>
   <receiver>VolumeCalculatorDialog</receiver>
   <slot>calculates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculates()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="bgLoSurface"/>
  <buttongroup name="bgUpSurface"/>
 </buttongroups>
</ui>
